### DISCUSSION TITLE
In dropdown on clicking the Trigger Button it gives a warning

### DISCUSSION THREAD

**Discussion by Riddhesh1009** [heroui-inc/heroui#1622] [Q&A]
I was using Dropdown in my nextjs13 project i am getting a console "Warning" that the key prop is spread into JSX is it intended behaviour or a bug??
![image](https://github.com/nextui-org/nextui/assets/74883681/d70b2891-34d4-4953-98d2-c3dc14150057)
If it is a bug than can I try solving it??

**Comment by jrgarciadev** [comment]
Which NextUI version do you have installed?

**Reply by Riddhesh1009** [reply]
NextUI 2.0.8
